<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAirlineTicketOrderManager</class>
 <widget class="QWidget" name="FrmAirlineTicketOrderManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{border:1px solid rgba(100, 100, 100,100);}
#frame{border-radius:10px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(223, 237, 233,100);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>560</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{width:140}
QTabBar::tab{height:40}</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>全部</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_all">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>511</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>待支付</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_all_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>511</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>已出票</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_all_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>511</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>已完成</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_all_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>511</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
